CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 6.0.0
+* BREAKING: Upgrades the underlying version of sidekiq. Will require changes to logging strategy;
+
+* Symbolized keys are no longer supported, stringy keys only
```

Comment:
I think we should probably phrase this as Sidekiq arguments need to be serialisable to JSON or you get a warning and reference: https://github.com/mperham/sidekiq/blob/main/Changes.md#640

Reply:
Done :)